libsql-server/Cargo.toml

```diff
@@ -38,6 +38,7 @@ jsonwebtoken = "9"
 libsql = { path = "../libsql/", optional = true }
 libsql_replication = { path = "../libsql-replication" }
 libsql-wal = { path = "../libsql-wal/" }
+libsql-storage = { path = "../libsql-storage" }
```

Comment:
can you make it a feature for now?

Reply:
yes, I have made it now.

Reply:
I need `libsql-server-devel` docker images with `durable-wal` enabled, so I have updated the github workflow: `server-pr-images`. But unfortunately it can be tested after merging only.